oh_staff_ui/tests.py

```diff
@@ -889,6 +901,27 @@ def setUpTestData(cls):
             type=ItemType.objects.get(type="Interview"),
             parent=cls.series_item,
         )
+        ItemLanguageUsage.objects.create(item=cls.interview_item, value=language)
+        # type_id values are taken from description-type-data.json fixture
+        # adminnote = 1
+        Description.objects.create(
+            item=cls.interview_item,
+            value="Abstract element",
+            type_id=1,
+        )
+        # adminnote = 4
+        Description.objects.create(
+            item=cls.interview_item,
+            value="Processing of interview:Process interview description as admin note",
```

Comment:
If diacritics / special characters are a concern, add a Description with some of them to test the output.